apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java

```diff
@@ -207,4 +207,6 @@
   
     public static final OfficialComponent GUAVA_CACHE = new OfficialComponent(114, "GuavaCache");
 
+    public static final OfficialComponent DRUID = new OfficialComponent(115, "druid");
```

Comment:
`druid` should be renamed as `alibaba druid`. As I checked, Druid is an Apache software foundation trademark, see https://druid.apache.org/.

I treat this as potential trademark conflict, we at least should keep it clear.

Reply:
I think what you said makes sense, and I will make changes.